packages/test-app/tests/integration/modifiers/get-element-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Modifier | get-element', function (hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
+  test('it renders', async function (assert) {
+    await render(hbs`<div {{get-element this.setRootElement}}></div>`);
+    assert.ok(true);
```

Comment:
Needs a real test here to ensure that `this.setRootElement` gets called. Also needs to test the assertion failure when a non-callback is passed. I didn't check, but I bet this test is currently throwing a warning in the console when it runs.

Reply:
Added a real set of tests, but see https://github.com/ilios/frontend/pull/8280#discussion_r1894261100 as to why I have not been able to test a "failing" assertion yet.